core/cmd/client_test.go

```diff
@@ -305,3 +310,17 @@ func TestFileSessionRequestBuilder(t *testing.T) {
 		})
 	}
 }
+
+func TestNewUserCache(t *testing.T) {
+
+	r, err := rand.Int(rand.Reader, big.NewInt(256*1024*1024))
```

Comment:
Unchecked err?

Reply:
thanks. done

Reply:
Did the linter flag this BTW? We might be omitting _test files or something - should reconsider that.

Reply:
it did not. i agree.